options/locale/locale_ru-RU.ini

```diff
@@ -754,6 +754,18 @@ settings.add_key_success=Новый ключ развертывания '%s' у
 settings.deploy_key_deletion=Удалить ключ развертывания
 settings.deploy_key_deletion_desc=Удаление ключа развертывания приведет к удалению всех связанных прав доступа к репозиторию. Вы хотите продолжить?
 settings.deploy_key_deletion_success=Ключ развертывания успешно удален!
+settings.branches=Ветки
```

Comment:
Since we switched to crowdin this have to be done within crowdin, otherwise we get into issues while we try to sync it.